src/home/swap-container/components/swap/index.tsx

```diff
@@ -959,6 +981,11 @@ const Swap = ({
     }
 
     setCreateStep(step);
+    if (step === 1) {
+      trackEvent('DCA - Continue to step 2', { fromAddress: from?.address, toAddress: to?.address });
+    } else {
+      trackEvent('DCA - Go back to step 1', { fromAddress: from?.address, toAddress: to?.address });
+    }
```

Comment:
Really noice to have this! We will finally understand how are users using our UI 🙏